lib/localparticipant.js

```diff
@@ -92,6 +100,14 @@ LocalParticipant.prototype._handleTrackSignalingEvents = function _handleTrackSi
       signaling.removeListener('stateChanged', stateChanged);
       self.removeListener('trackAdded', localTrackAdded);
       self.removeListener('trackRemoved', localTrackRemoved);
+
+      if (self._tracksToStop.size) {
```

Comment:
I think a single log message if there are any LocalTracks to stop might be nicer. Each call to `LocalTrack#stop` is going to log, anyway. So the difference is between

```
[LocalParticipant] LocalParticipant disconnected. Stopping automatically-acquired LocalTracks
[LocalTrack 1] Stopping
[LocalTrack 2] Stopping
```

And

```
[LocalParticipant] LocalParticipant disconnected.
[LocalParticipant] Stopping automatically-acquired LocalTrack: [LocalTrack 1]
[LocalTrack 1] Stopping
[LocalParticipant] Stopping automatically-acquired LocalTrack: [LocalTrack 2]
[LocalTrack 2] Stopping
```

WDYT?